clients/admin-ui/src/features/privacy-experience/ConfigurePrivacyExperience.tsx

```diff
@@ -136,13 +138,36 @@ const ConfigurePrivacyExperience = ({
     TranslationWithLanguageName | undefined
   >(undefined);
 
-  const handleNewTranslationSelected = (translation: ExperienceTranslation) => {
+  const [translationIsOOB, setTranslationIsOOB] = useState<boolean>(false);
+
+  const getOOBTranslation = (language: SupportedLanguage) => {
```

Comment:
nit: you only use this function in the body of `handleCreateNewTranslation`, so I feel like you should inline it there instead and provide a quick comment to explain why this exists 👍